### PR TITLE
HTML To PDF By Pascalcase

### PR DESCRIPTION
**Pull request by VikrantUpadhyay1**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

**Comment by VikrantUpadhyay1** [comment]
> @VikrantUpadhyay1 please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following information.
> 
> ```
> @microsoft-github-policy-service agree [company="{your company}"]
> ```
> 
> > Options:
> > 
> > * (default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > 
> > * (when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree company="Microsoft"
> > ```
> 
> Contributor License Agreement

@microsoft-github-policy-service agree [company="Pascalcase Software Pvt.Ltd"]

**Comment by VikrantUpadhyay1** [comment]
> @VikrantUpadhyay1 the command you issued was incorrect. Please try again.
> 
> Examples are:
> 
> ```
> @microsoft-github-policy-service agree
> ```
> 
> and
> 
> ```
> @microsoft-github-policy-service agree company="your company"
> ```

@microsoft-github-policy-service agree
@microsoft-github-policy-service agree company="Pascalcase Software Pvt.Ltd"

**Comment by VikrantUpadhyay1** [comment]
@microsoft-github-policy-service agree company="Pascalcase Software Pvt.Ltd"

**Comment by VikrantUpadhyay1** [comment]
> @microsoft-github-policy-service agree company="Pascalcase Software Pvt.Ltd"


@microsoft-github-policy-service agree company="Pascalcase Software Pvt.Ltd"

File: certified-connectors/HTML To PDF By Pascalcase/settings.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (Amjed-Ayoub) [inline certified-connectors/HTML To PDF By Pascalcase/settings.json]: @VikrantUpadhyay1 Please remove the settings.json file and the Icon.png from this pull request, as they are not required in the open source repository.

File: certified-connectors/HTML To PDF By Pascalcase/apiProperties.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+  "properties": {
+    "connectionParameters": {},
+    "iconBrandColor": "#18186d",
+    "capabilities": [],
+    "publisher": "Vikrant Upadhyay"
```
Comment (Amjed-Ayoub) [inline certified-connectors/HTML To PDF By Pascalcase/apiProperties.json]: @VikrantUpadhyay1 Please add the "stackOwner" property. Please see our template [here](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/apiProperties.json). 
```
"stackOwner": "Name of the owner of the service or product."
```

Comment (Amjed-Ayoub) [inline certified-connectors/HTML To PDF By Pascalcase/apiProperties.json]: @VikrantUpadhyay1 for all our Verified Publishers/API Owners we ask to have the "publisher" and the "stackOwner" set to the same company name. 
```
"publisher": "Name of company or organization.",
"stackOwner": "Name of the owner of the service or product."
```

**Review by Amjed-Ayoub — CHANGES_REQUESTED** [review]

File: certified-connectors/HTML To PDF By Pascalcase/apiProperties.json (hunk 2 above)
Comment (VikrantUpadhyay1) [reply certified-connectors/HTML To PDF By Pascalcase/apiProperties.json]: @Amjed-Ayoub thank you for your feedback, I have made all the changes which were asked by you.

**Review by Amjed-Ayoub — APPROVED** [review]
Hello @VikrantUpadhyay1 ,
I hope you are doing well.
Congratulations, your pull request is approved and merged. Please [submit the files to ISV Studio](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) or [update your submission](https://learn.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#update-submission-prior-to-certification) using your latest commit ID, please allow us up to 1-2 weeks to review your submission, and our Engineers will notify you in the “Activity Control” area in ISV Studio. Please [make sure that the icon meets all the requirements](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#design-an-icon-for-your-connector), that it’s size is 230x230, icon background is not white or transparent and it matches the iconBrandColor property in apiProperties.json file.
After making the submission in ISV Studio, please attach an intro.md file in the activity control section. This file is required and it's different from readme.md. You can [find an example of intro.md file on this page](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) at step 6.
Please [Create an environment in the Preview region](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#create-an-environment-in-the-preview-region). You will use this environment later to [test your connector](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#test-your-connector) after Microsoft is done with the functional verification for your connector. 
We expect all tests to be completed within 1-2 weeks.
If your connector passes all tests, the deployment process begins, and it typically takes up to 3 to 4 weeks to deploy your connector to all of our regions.
Please let me know if you have any questions or run into any problems while creating or updating your submission.
Thank you very much for working with us.

**Comment by VikrantUpadhyay1** [comment]
@Amjed-Ayoub The connector just failed on Azure pipeline, could you help me understand whats the cause and is it going to affect the connector?
image

**Comment by Amjed-Ayoub** [comment]
@VikrantUpadhyay1 I just rerun the check, please wait. I will keep you posted.

File: certified-connectors/HTML To PDF By Pascalcase/apiProperties.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+  "properties": {
+    "connectionParameters": {},
+    "iconBrandColor": "#18186d",
+    "capabilities": [],
+    "publisher": "Pascalcase Software Pvt.Ltd"
```
Comment (Amjed-Ayoub) [inline certified-connectors/HTML To PDF By Pascalcase/apiProperties.json:6]: @VikrantUpadhyay1 Please add a comma "," at the end of the line 6

**Comment by VikrantUpadhyay1** [comment]
@Amjed-Ayoub I really appreciate the prompt response I will wait for your feedback.

File: certified-connectors/HTML To PDF By Pascalcase/apiDefinition.swagger.json [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [],
+  "paths": {
+    "/": {
+      "post": {
+        "responses": {
+          "default": {
```
Comment (Amjed-Ayoub) [inline certified-connectors/HTML To PDF By Pascalcase/apiDefinition.swagger.json:26]: @VikrantUpadhyay1 It is not allowed to have a default response only. Please define a proper response. Please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/coding-standards#responses).

File: certified-connectors/HTML To PDF By Pascalcase/apiDefinition.swagger.json [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+            "description": "default",
+            "schema": {}
```
Comment (Amjed-Ayoub) [inline certified-connectors/HTML To PDF By Pascalcase/apiDefinition.swagger.json:28]: @VikrantUpadhyay1 empty schema is not allowed, please define a proper schema.

**Comment by Amjed-Ayoub** [comment]
@VikrantUpadhyay1 Please accept my apologies for not seeing this before. Please create a new PR.

**Comment by VikrantUpadhyay1** [comment]
@Amjed-Ayoub Do I need to delete the existing connector repo? or do I need to update the same?

**Comment by Amjed-Ayoub** [comment]
> @Amjed-Ayoub Do I need to delete the existing connector repo? or do I need to update the same?

@VikrantUpadhyay1 yes please update the same.

**Pull request merged**